Sources/ObjCSupport/LOKBaseLayout.swift

```diff
@@ -0,0 +1,45 @@
+// Copyright 2018 LinkedIn Corp.
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+
+import CoreGraphics
+
+public class LOKBaseLayout: NSObject, LOKLayout {
+    let layout: Layout
+
+    init(layout: Layout) {
```

Comment:
We keep all `init` internally for wrapper use?

Reply:
Yes.